sympy/polys/tests/test_polytools.py

```diff
@@ -1900,6 +1900,8 @@ def test_gcd_list():
     gcd = gcd_list([], x, polys=True)
     assert gcd.is_Poly and gcd.is_zero
 
+    assert gcd_list([sqrt(2), -sqrt(2)]) == sqrt(2)
```

Comment:
This one passes for me in master but the reverse fails. Perhaps just include both:
```
s = sqrt(2)
assert gcd_list([a, -a]) == gcd_list([-a, a]) == s
```

Reply:
Yeah, the reverse is also passing for me, I'll add those changes you suggested.